### PR TITLE
Add deodr

### PR DESCRIPTION
**Pull request by oblute**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by oblute** [comment]
@ndmaxar @rluria14 agree to maintain?

File: recipes/deodr/LICENSE [hunk 1]
```diff
@@ -0,0 +1,25 @@
+BSD 2-Clause License
```
Comment (synapticarbors) [inline recipes/deodr/LICENSE:1]: I added a PR to address the manually packaged license file at https://github.com/martinResearch/DEODR/pull/34

File: recipes/deodr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  url: https://github.com/martinResearch/DEODR/releases/download/v0.1.14/deodr-0.1.14.tar.gz
+  sha256: 8e559df8e119d949994213a1f1ae77210def7ccfa248dc8ceeabd9505e389772
+  patches:
+    # Currently the required CXX file for build isn't packaged in the release, so generated here.
+    - setup_CXX.patch
+
+build:
+  number: 0
+  script:
+    - {{ PYTHON }} setup.py build_ext --inplace
```
Comment (synapticarbors) [inline recipes/deodr/meta.yaml:18]: Do we need to do the `build_ext` separately? I would think the `pip install` on the next line would be sufficient to build the extension modules.

File: recipes/deodr/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - cython
+    - numpy
+    - scipy
+  run:
+    - python
+    - numpy
```
Comment (synapticarbors) [inline recipes/deodr/meta.yaml:33]: If you're building against numpy see:

https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

File: recipes/deodr/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
+{% set name = "deodr" %}
+{% set version = "0.1.14" %}
```
Comment (synapticarbors) [inline recipes/deodr/meta.yaml:2]: It looks like `0.1.15` is up on `pypi.io`, so you might consider building from that tarball.

**Comment by oblute** [comment]
@synapticarbors i built from the newly released version and made your changes

**Pull request merged**